README.md

```diff
@@ -343,52 +301,47 @@ is necessary in order to allow pipelining of messages before the exact
 destination is known (i.e. before the delegated promise is resolved).
 
 ```js
-Promise.eventualGet(target, prop); // Promise<result>
-Promise.eventualGetOnly(target, prop); // undefined
-```
-
-```js
-Promise.eventualApply(target, [...args]); // Promise<result>
-Promise.eventualApplyOnly(target, [...args]); // undefined
+Promise.delegated.eventualGet(target, prop, opts = {}); // Promise<result>
+Promise.delegated.eventualApply(target, [...args], opts = {}); // Promise<result>
 ```
 
 The `eventualSend` call combines property lookup with function application in
-order to distinguish them from an `eventualGet` whose value is separately inspected, and
-for the handler to be able to bundle the two operations as a single message.
+order to distinguish them from an `eventualGet` whose value is separately
+inspected, and for the handler to be able to bundle the two operations as a
+single message.
 
 ```js
-Promise.eventualSend(target, prop, args); // Promise<result>
-Promise.eventualSendOnly(target, prop, args); // undefined
+Promise.delegated.eventualSend(target, prop, args, opts = {}); // Promise<result>
 ```
 
 ## Platform Support
 
-All the above behavior, as described so far, is implemented in the [Eventual
-Send Shim](https://github.com/Agoric/eventual-send) (TODO update to proposed
-API).  However, there is one critical behavior that we specify, that can easily
-be provided by a conforming platform, but is infeasible to emulate on top of
+All the above behavior, as described so far, will be implemented in the [Eventual
```

Comment:
(but it's close, right?) No change suggested.

Reply:
Yes, just needs some refactoring.